frontend/src/pages/App.tsx

```diff
@@ -42,6 +42,7 @@ i18n.use(detector)
 const axiosInstance = axios.create();
 axiosInstance.defaults.baseURL = import.meta.env.VITE_BACKEND_URL;
 axiosInstance.defaults.timeout = 6000;
+axiosInstance.defaults.withCredentials = true;
```

Comment:
This should not be implemented because only some requests require authorization.

Reply:
No.  This is required due to the backend gaining user information using the cookie. If this is not set, the backend cannot retrieve the information, because the cookie is not sent.

Reply:
It is set per request. That way it can be controlled when to send the cookies.

Reply:
Now it was removed. The requests that requires the cookie will be defined.